src/test/java/org/jenkinsci/plugins/docker/workflow/DockerDSLTest.java

```diff
@@ -149,7 +149,7 @@ private static void grep(File dir, String text, String prefix, Set<String> match
                 assumeDocker();
                 WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
                 p.setDefinition(new CpsFlowDefinition(
-                        "docker.image('maven:latest').inside {\n" +
+                        "docker.image('maven:latest').overrideEntrypoint {\n" +
```

Comment:
In practice this is just going to be annoying, and in particular will not work from `pipeline-model-definition`’s Docker agent unless additional flags are added to that.